taxi/urls.py

```diff
@@ -46,6 +51,27 @@
     path(
         "drivers/<int:pk>/", DriverDetailView.as_view(), name="driver-detail"
     ),
+    path("drivers/create/", DriverCreateView.as_view(), name="driver-create"),
+    path(
+        "drivers/<int:pk>/update",
```

Comment:
you forgot to add "/" at the end
Everything else looks good as for me)

Reply:
I made same mistake 😂